docs/getting_started/getting_started.rst

```diff
@@ -26,6 +26,12 @@ Why use :func:`rioxarray.open_rasterio` instead of `xarray.open_rasterio`?
 5. It adds the coordinate axis CF metadata.
 6. It loads raster metadata into the attributes.
 
+Starting with xarray 0.18.0 rioxarray enables passing `engine="rasterio"` to all xarray open functions:
```

Comment:
```suggestion
rioxarray 0.4+ enables passing `engine="rasterio"` to ``xarray.open_dataset``:
```

Reply:
>  xarray 0.18.0

Shouldn't this work with 0.17+ or did I miss something?

Reply:
> rioxarray 0.4+ 

I modified it to this since it will have the pin `xarray>=0.17`

Reply:
The new backend API is now in `master` and will be in the next release that should 0.18.0.

So you need rioxarray 0.4+ and xarray 0.18+.